OpenXmlPowerTools/HtmlToWmlCssParser.cs

```diff
@@ -1143,113 +961,82 @@ public enum CssTagType
-        }
+        public char? Separator { get; set; }
 
         public string SeparatorChar
         {
-            get => m_separator.HasValue ? m_separator.Value.ToString() : null;
-            set => m_separator = !string.IsNullOrEmpty(value) ? value[0] : '\0';
+            get => Separator.HasValue ? Separator.Value.ToString() : null;
+            set => Separator = !string.IsNullOrEmpty(value) ? value[0] : '\0';
         }
 
-        public char? Sign
-        {
-            get => m_sign;
-            set => m_sign = value;
-        }
+        public char? Sign { get; set; }
 
         public string SignChar
         {
-            get => m_sign.HasValue ? m_sign.Value.ToString() : null;
-            set => m_sign = !string.IsNullOrEmpty(value) ? value[0] : '\0';
+            get => Sign.HasValue ? Sign.Value.ToString() : null;
+            set => Sign = !string.IsNullOrEmpty(value) ? value[0] : '\0';
         }
 
-        public CssTermType Type
-        {
-            get => m_type;
-            set => m_type = value;
-        }
+        public CssTermType Type { get; set; }
 
-        public string Value
-        {
-            get => m_val;
-            set => m_val = value;
-        }
+        public string Value { get; set; }
 
-        public CssUnit? Unit
-        {
-            get => m_unit;
-            set => m_unit = value;
-        }
+        public CssUnit? Unit { get; set; }
 
         public string UnitString
         {
             get
             {
-                if (m_unit.HasValue)
+                if (Unit.HasValue)
                 {
-                    return m_unit.ToString();
+                    return Unit.ToString();
                 }
                 else
                 {
                     return null;
                 }
             }
-            set => m_unit = (CssUnit)Enum.Parse(typeof(CssUnit), value);
+            set => Unit = (CssUnit)Enum.Parse(typeof(CssUnit), value);
         }
 
-        public CssFunction Function
-        {
-            get => m_function;
-            set => m_function = value;
-        }
+        public CssFunction Function { get; set; }
 
         public override string ToString()
         {
             var sb = new StringBuilder();
 
-            if (m_type == CssTermType.Function)
+            if (Type == CssTermType.Function)
             {
-                sb.Append(m_function.ToString());
+                sb.Append(Function.ToString());
             }
-            else if (m_type == CssTermType.Url)
+            else if (Type == CssTermType.Url)
             {
-                sb.AppendFormat("url('{0}')", m_val);
+                sb.AppendFormat("url('{0}')", Value);
             }
-            else if (m_type == CssTermType.Unicode)
+            else if (Type == CssTermType.Unicode)
             {
-                sb.AppendFormat("U\\{0}", m_val.ToUpper());
+                sb.AppendFormat("U\\{0}", Value.ToUpper());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=5308324)